Update creating_a_webhook.md (#10009)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/webhooks/creating_a_webhook.md

Braze retries the above status codes up to five times within 30 minutes using exponential backoff. If we can't reach your endpoint, retries may be spread over a 24-hour period.<br><br>Each webhook is allowed 90 seconds before it times out.
{% endalert %}

> #### Troubleshooting and additional error details
> For detailed explanations, troubleshooting steps, and guidance on resolving specific webhook errors, please refer to the [Troubleshooting webhook and Connected Content requests]({{site.baseurl}}/help/help_articles/api/webhook_connected_content_errors/) help article.
> 
> The help article covers commons webhook errors, explains Braze's unhealthy host detection system, and describes how Braze provides error notifications through automated emails and additional logging in Braze Currents.

Reviewer: ```suggestion
#### Troubleshooting and additional error details

For detailed explanations, troubleshooting steps, and guidance on resolving specific webhook errors, refer to [Troubleshooting webhook and Connected Content requests]({{site.baseurl}}/help/help_articles/api/webhook_connected_content_errors/). You'll also find more explanations on how our unhealthy host detection system works and how Braze provides error notifications through automated emails and additional logging in Braze Currents.
```
